### PR TITLE
LPS-153687 Add collaborators to the Info Panel after sharing an asset

### PR DESCRIPTION
**Pull request by cgoncas**
### Motivation

As a content marketer, I want to see all my collaborators after sharing an asset, so that I can easily view and manage my collaborators on a specific asset.

### Proposed Solution

This solution use the same approach described in https://github.com/liferay-tango/liferay-portal/pull/2612.

We have created a sharing_collaborators_button.jsp that includes the sharing:collaborators taglib, so the sharing collaborators is rendered using a fetch, as we do with the content performance panel.

### CODE REVIEW

**Comment by cgoncas** [comment]
Hey @beltranrengifo and @Stephy6 !!!

I have create the collaborators JSP, I need to rename the SHARING_COLLABORATORS_BUTTON as SHARING_COLLABORATORS, but with the current PR, you can use the **_fetchSharingContactsButtonURL_** to obtain the JSP URL.

I have include it only for blogs.

Thanks!!

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java [hunk 1]
```diff
@@ -137,6 +137,10 @@ protected void doServeResource(
 					_toString(contentDashboardItem.getCreateDate())
 				).put(
 					"description", contentDashboardItem.getDescription(locale)
+				).put(
+					"fetchSharingContactsButtonURL",
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java:141]: this naming is a bit weird to me 🤔 
`fetchManageCollaboratorsButtonURL` could do it better?

**Comment by beltranrengifo** [comment]
@Stephy6 We'll need to implement this also for Documents, as you are doing with the Share btn

**Comment by beltranrengifo** [comment]
hey @cgoncas I've rebase with the ADD_SHARE_BTN pr, as I need those changes too.
now i need to push -f, is it okay with you? thxs

**Comment by beltranrengifo** [comment]
@Stephy6 I've rebase with the Share_btn pr and force pushed

**Comment by beltranrengifo** [comment]
force pushing with some changes

**Comment by beltranrengifo** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
Hey @Stephy6 and @beltranrengifo !!

I have cherry picked all the backend changes and try to send it here: https://github.com/liferay-tango/liferay-portal/pull/2636

I let you to do the same with the frontend changes.

In the end, following Liferay handbook I have renamed the fetch as: fetchSharingCollaboratorsURL (because is the name of the action/taglib ...)

Thanks!!

**Comment by cgoncas** [comment]
Closing, all the code have been merged on master.

**Pull request closed without merging**